src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -85,6 +86,10 @@ public abstract class HttpClientHandlerTest : HttpClientTestBase
             new object[] { 307, "GET", "GET" },
             new object[] { 307, "POST", "POST" },
             new object[] { 307, "HEAD", "HEAD" },
+
+            new object[] { 308, "GET", "GET" },
+            new object[] { 308, "POST", "POST" },
+            new object[] { 308, "HEAD", "HEAD" },
```

Comment:
I'll be interested to see if this passes on NetfxHandler and CurlHandler.

Reply:
And UapHandler, since:
https://github.com/dotnet/corefx/blob/015ee21f6fe1eadad478d8c9409c95cf98c2b121/src/System.Net.Http/src/uap/System/Net/HttpHandlerToFilter.cs#L115-L122

Reply:
Agreed. I think it's possible that it will also fail on win7. If so though I think we should keep the change and disable the test on those platforms.